src/components/group/GroupForm.js

```diff
@@ -93,7 +93,7 @@ export default class GroupForm extends Component {
     const {gid, session, bucket, group, formData, deleteGroup} = this.props;
     const creation = !formData;
     const hasWriteAccess = creation ? canCreateGroup(session, bucket)
-                                    : canEditGroup(session, group);
+                                    : canEditGroup(session, group, bucket);
```

Comment:
A group depends on a bucketThe parent of a group being a bucket, I suggest the `canEditGroup(session, bucket, group)` instead